src/app/_classes/address.ts

```diff
@@ -19,4 +22,14 @@ export class Address {
       !!this.stateProvince ||
       !!this.zipPostalCode;
   }
+
+  public isComplete(countryCode: CountryCode, states: StateCode[]): boolean {
+    return !!this.street &&
+    !!this.city &&
+    !!this.country &&
+    (states.length > 0 ? !!this.stateProvince : true) &&
+    (countryCode ?
+      (countryCode.zipRequired ? !!this.zipPostalCode : true) :
+      false);
+  }
```

Comment:
Wasn't this part of stuff @CelloH3r0 did? Or is this something you added?

I just tried to do main -> dev and I'm not seeing it... so just wondering what this is

Reply:
ohh, looks like you just moved this to be an aspect of the address... didn't scroll down to read the whole PR

Reply:
I refactored some stuff by moving them into the actual objects. I don't remember what Lucas did though. This project has been going on too long for me to remember... I'm not pepperidge farms.